### PR TITLE
Display databases and classes from dependency projects in the source option dropdown

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

Fixed https://github.com/finos/legend-studio/issues/890

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: .changeset/gold-shirts-switch.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (akphi) [inline .changeset/gold-shirts-switch.md:1]: Let's be mindful of changeset granuality. This change should be split across 2 files. The message you wrote is relevant only for Studio

File: .changeset/gold-shirts-switch.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+'@finos/legend-graph': patch
+'@finos/legend-studio': patch
+---
+
+Display databases and classes from dependency projects in the source options dropdown.
```
Comment (akphi) [inline .changeset/gold-shirts-switch.md:6]: Should also add `in mapping editor`

**Review by akphi — CHANGES_REQUESTED** [review]

File: .changeset/shy-rockets-melt.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+---
+
+Update the dependencyManager to include databases.
```
Comment (MauricioUyaguari) [inline .changeset/shy-rockets-melt.md:5]: @akphi  I think a change this small probably doesn't need a changeset log. Thoughts?

File: .changeset/gold-shirts-switch.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+Display databases and classes from dependency projects in the source options dropdown in the mapping editor.
```
Comment (akphi) [inline .changeset/gold-shirts-switch.md:5]:
```suggestion
Display `database tables/views` and `classes` from dependency projects in the source selector dropdown in the mapping editor.
```

**Review by akphi — APPROVED** [review]

**Pull request merged**